seed-job/template.yaml

```diff
@@ -99,4 +101,7 @@ parameters:
   displayName: Job ID
   name: JOB_ID
   required: true
+- description: registry_url is the url of the registry, into which the push is supposed to happen.
```

Comment:
It's clear that we're giving providing description of `REGISTRY_URL` variable. So we can just make it something like: `URL of the registry to which image is to be pushed`.

Also, please follow the convention that's followed in rest of the file and provide a `displayName`. I'm looking for something like:
```
- description: "URL of the registry to which image is to be pushed"
  name: REGISTRY_URL
  displayName: Registry URL
  required: true
```